client/app/lib/redux/services/payment.coffee

```diff
@@ -36,6 +36,8 @@ exports.fetchSubscription = fetchSubscription = pickData ->
   client.get Endpoints.SubscriptionGet
 
 exports.createSubscription = createSubscription = pickData (params = {}) ->
+  if not params.trialEnd
+    params.trialEnd = (new Date()).getTime() + (30 * 24 * 60 * 60 * 1000)
```

Comment:
Whoa, i had no idea about this :D I guess it first applies `toString` and then `+` makes it a number?

But my comment was mainly because I felt that `Date.now()` is more readable than `(new Date()).getTime()`